README.md

```diff
@@ -7,6 +7,14 @@
 - [Marketing automation platform wich have ecommerce integration](#marketing-automation-platform-wich-have-ecommerce-integration)
 - [Automation Email Marketing platform](#automation-email-marketing-platform)
 - [Tools for effective works with e-mail](#tools-for-effective-works-with-e-mail)
+- [Strategy](#strategy)
+- [Similar companies](#similar-companies)
+- [Building a list](#building-a-list)
+- [Data Driven Email Mark](#data-driven-email-mark)

```

Comment:
can you change it to "Data Driven Email Marketing"

Reply:
Done